### PR TITLE
improve docker documentation in readme

### PR DESCRIPTION
**Pull request by juan-apa**
#### Board:
 - references #238
---
#### Description:
Improve docker usage documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,10 +38,25 @@ This template comes with:
 
 ## How to use with docker
 
-1. Have docker and docker-compose installed
-1. build and run the application with `docker-compose up`
-1. Run `docker-compose exec web bash -c "rails db:create db:migrate"`
-1. You can now try your REST services!
+1. Have `docker` and `docker-compose` installed (You can check this by doing `docker -v` and `docker-compose -v`)
+2. Modify the following lines in the `database.yml` file:
```
Comment (santib) [inline README.md:42]: you can leave `1.` and it will take the correct number automatically

Comment (juan-apa) [reply README.md:42]: If I leave it with `1.`, the numbering restarts after the code block :eyes:

**Review by martinjaimem — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -38,10 +38,25 @@ This template comes with:
...
+  default: &default
+    adapter: postgresql
+    encoding: unicode
+    pool: 5
+    username: postgres
+    password: postgres
+    host: db
+    port: 5432
+  ```
+3. Generate a secret key for the app by running `docker-compose run --rm --entrypoint="" web rake secret`, copy it and add it in your environment variables.
```
Comment (sebalopez) [inline README.md:53]: you don't necessarily have to do this through docker-compose, but this works too

**Review by sebalopez — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (juan-apa) [reply README.md:53]: you mean creating the secret?

**Review by santib — APPROVED** [review]

**Pull request merged**